local-interchaintest/src/tests/two_party_pol/two_party_pol_native.rs

```diff
@@ -258,11 +250,11 @@ pub fn test_two_party_pol_native(test_ctx: &mut TestContext) -> Result<(), Local
         &pool_addr,
         ACC_0_KEY,
         &serde_json::to_string(&provide_liquidity_msg).unwrap(),
-        &format!("--amount {uatom_contribution_amount}{atom_on_neutron_denom},{untrn_contribution_amount}{neutron_denom} {EXECUTE_FLAGS}"),
+        &format!("--amount {uatom_contribution_amount}{atom_on_neutron},{untrn_contribution_amount}{neutron_denom} {EXECUTE_FLAGS}"),
     ).unwrap();
     thread::sleep(Duration::from_secs(3));
 
-    // Instantiate the covenant
+    // Instantiate the covenants
```

Comment:
should this comment be later? seems like following calls are for getting the code ids, not for instantiating the covenant